src/app/auth/sign-up/sign-up.component.scss

```diff
@@ -0,0 +1,52 @@
+@import '../../landing/landing.component';
+
+body {
+    background-color: #18BC9C;
+    margin: -100;
+}
+.form-signin {
+  max-width: 330px;
+  padding: 15px;
+  margin: 0 auto;
+}
+.form-signin .form-signin-heading,
+.form-signin .checkbox {
+  margin-bottom: 10px;
+}
+.form-signin .checkbox {
+  font-weight: normal;
+}
+.form-signin .form-control {
+  position: relative;
+  height: auto;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  padding: 10px;
+  font-size: 16px;
+}
+.form-signin .form-control:focus {
+  z-index: 2;
+}
+.form-signin input[type="email"] {
+  margin-bottom: -1px;
+  border-bottom-right-radius: 0;
+  border-bottom-left-radius: 0;
+}
+.form-signin input[type="password"] {
+  margin-bottom: 10px;
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+}
+
+.form-signin-header {
+  color: white;
+}
+
+button {
+  cursor: pointer;
```

Comment:
We might want to move global stuff like this to a our global styling file. Though I guess it's not necessary just now since we don't really have a solid style yet anyways,